plugin/src/main/java/org/jfrog/idea/xray/persistency/types/Issue.java

```diff
@@ -14,8 +14,6 @@
     public String description;
     public String issueType = "N/A";
     public String provider;
-    @Deprecated
-    public String sevirity;
```

Comment:
Are you sure we can remove this deprecated property? People may still be using an old version the plugin, with this property in the cache, and upgrading to this version will cause issues.
What do you think?

Reply:
I thought its unlikely that people are still using the version from 2017 with Intellij's auto updates. I'll revert it to be on the safe side.